packages/laconia-adapter-api/src/ApiGatewayNameMappingErrorConverter.js

```diff
@@ -6,7 +6,7 @@ const getMappingEntries = mappings =>
 const getMappingResponse = (mappings, error) => {
   let mappingResponse = {};
   for (const [errorRegex, mapping] of getMappingEntries(mappings)) {
-    if (error.name.match(errorRegex)) {
+    if (error.name.match(errorRegex) || error.message.match(errorRegex)) {
```

Comment:
Do you think the order matters? Probably not, but just wondering if there's a case where the `error.name` matches for one status code pattern, and the `error.message` matches for another status code pattern.

Reply:
Your scenario is valid but I think we can specify that `error.name` takes precedence over `error.message`. I'm happy to add this to the docs.

Reply:
Good solution, if it makes `name` takes precedence over `message` it won't change actual expected behavior, for most of cases.
It would not be a breaking change, will not require a feature flag nor a version bump.